Configuration.props

```diff
@@ -19,7 +19,8 @@
     <AndroidMxeInstallPrefix Condition=" '$(AndroidMxeInstallPrefix)' == '' ">$(AndroidToolchainDirectory)\mxe</AndroidMxeInstallPrefix>
     <AndroidSdkDirectory>$(AndroidToolchainDirectory)\sdk</AndroidSdkDirectory>
     <AndroidNdkDirectory>$(AndroidToolchainDirectory)\ndk</AndroidNdkDirectory>
-    <AndroidSupportedAbis Condition=" '$(AndroidSupportedAbis)' == '' ">host-$(HostOS):armeabi-v7a</AndroidSupportedAbis>
+    <AndroidSupportedDeviceAbis>armeabi-v7a</AndroidSupportedDeviceAbis>
```

Comment:
AndroidSupportedAbis is, potentially, separated with `:` and that requires extra processing. Also, it contains the `host-$(HostOS)` entry which needs to be exclude as well - more extra processing. So it's better to split AndroidSupportedAbis in the way I did to make things simpler (AndroidSupportedDeviveAbis will be separated with `;` - no extra processing necessary most of the time, with the exception being the `ndk-build` invocation)